service-fabric-secure-cluster-5-node-1-nodetype/metadata.json

```diff
@@ -2,10 +2,9 @@
   "$schema": "https://aka.ms/azure-quickstart-templates-metadata-schema#",
   "type": "QuickStart",
   "itemDisplayName": "Deploy a 5 Node Secure Cluster",
-  "description": "This template allows you to deploy a secure 5 node Service Fabric Cluster running Windows Server 2016 Datacenter on a Standard_D2 Size VMSS.",
+  "description": "This template allows you to deploy a secure 5 node Service Fabric Cluster running Windows Server 2019 Datacenter on a Standard_D2_v2 Size VMSS.",
   "summary": "This template takes a minimum amount of parameters and deploys 5 Node Service Fabric Cluster",
   "githubUsername": "chackdan",
```

Comment:
Could we update githubUsername to **erikadoyle** ? Thanks!

Reply:
Fixed.

Reply:
@erikadoyle I noticed the [metadata](https://github.com/azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md#metadatajson) guidance states `githubUsername` is for the original template author and shouldn't be changed. I'll revert the change.